src/hotspot/share/cds/archiveBuilder.cpp

```diff
@@ -907,9 +907,9 @@ void ArchiveBuilder::make_klasses_shareable() {
   log_info(cds)("      unregistered     " STATS_FORMAT, STATS_PARAMS(unregistered_klasses));
   log_info(cds)("      (hidden)         " STATS_FORMAT, STATS_PARAMS(hidden_klasses));
   log_info(cds)("      (unlinked)       " STATS_FORMAT ", boot = %d, plat = %d, app = %d, unreg = %d",
-                                                              STATS_PARAMS(unlinked_klasses),
-                                                              boot_unlinked, platform_unlinked,
-                                                              app_unlinked, unreg_unlinked);
+                                          STATS_PARAMS(unlinked_klasses),
+                                          boot_unlinked, platform_unlinked,
+                                          app_unlinked, unreg_unlinked);
```

Comment:
This indentation still looks weird. Arguments on newlines should be aligned.

Reply:
I tried to separate the "types" from the "values". I think this makes it easy to see how many types there are.

Reply:
Sorry I don't follow. This is just like a printf call

Reply:
I changed it according to your suggestion.